rocketpy/__init__.py

```diff
@@ -23,14 +15,26 @@
     LiquidMotor,
     MassBasedTank,
     MassFlowRateBasedTank,
+    Motor,
     SolidMotor,
     SphericalTank,
     Tank,
     TankGeometry,
     UllageBasedTank,
 )
-from .Parachute import Parachute
 from .plots import *
 from .prints import *
```

Comment:
I don't see much reason to expose as a to level import all the prints and plots classes. They have their names already "protected" (underscore) and I believe their usage as a standalone call is already contemplated by the `all_info` methods.

I tested `pytest` locally and simply removing these lines does not break any tests, as expected.

Reply:
Agreed, 

solved in my last commit.